app/views/letters/_header.html.erb

```diff
@@ -17,7 +17,7 @@
     <p><%= sender_city || department.capital %>, le <%= Date.today.strftime('%d/%m/%Y') %></p>
     <br/>
     <p>À l'attention de</p>
-    <br/><br/>
+    <br/><br/><br/>
```

Comment:
2064 be like 
```html
<br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/>
```

Reply:
🤷‍♂️😂

Reply:
un jour je mesurerai en pixel et je ferai une classe css. Un jour...